internal/configs/configurator.go

```diff
@@ -480,6 +574,10 @@ func (cnf *Configurator) addOrUpdateTransportServer(transportServerEx *Transport
 		return fmt.Errorf("Error generating TransportServer config %v: %v", name, err)
 	}
 
+	if (cnf.isPlus && cnf.isPrometheusEnabled) || cnf.isLatencyMetricsEnabled {
```

Comment:
we don't need to check for ` cnf.isLatencyMetricsEnabled`, because the latency collector doesn't support TransportServer